docs/modules/ROOT/pages/building.adoc

```diff
@@ -35,31 +35,33 @@ $ helm upgrade secret-operator deploy/helm/secret-operator \
 
 == Nix
 
-To build and deploy to the active Kind cluster, run:
+To build and deploy to the active Kind cluster, run the following bash script:
 
 [source,console]
 ----
 $ echo Building with Nix
 # Ensure that all submodules are up-to-date
 $ git submodule update --recursive --init
-# Ensure that the Cargo.lock is up-to-date
-# This is not required if you use a tool that invokes Cargo regularly anyway, such as Rust-Analyzer
-$ cargo generate-lockfile
 # Use crate2nix (https://github.com/kolloch/crate2nix) to convert Cargo.lock into a Nix derivation
-$ nix run -f . crate2nix generate 
+$ nix run -f . crate2nix generate
 # Build the Docker images
+# A custom Docker repository can be specified by appending `--argstr dockerName my-custom-repository`
```

Comment:
An image name could also be included in the example to make it clear that it is not appended automatically, e.g. `--argstr dockerName my-custom-repository/my-directory/secret-operator`.

Reply:
de76f931c1a3a44573722340da37ca62707c8e31